xbout/plotting/utils.py

```diff
@@ -41,26 +41,29 @@ def _decompose_regions(da):
     j11, j12, j21, j22, ix1, ix2, nin, _, ny, y_boundary_guards = _get_seps(da)
     regions = []
 
+    x, y = da.dims[-2:]
```

Comment:
Might be able to do this better, without relying on order of dimensions, if #68 is merged.